packages/redux/action-creator-dispatcher/mixin.runtime-common.js

```diff
@@ -69,7 +69,7 @@ class ReduxActionCreatorRuntimeMixin extends Mixin {
         if (!match) {
           return Promise.resolve();
         }
-        return store.dispatch(action(match.params));
+        return store.dispatch(action(match.params, location));
```

Comment:
In order to stay open to future API additions, I'd suggest to provide all **`actionCreators` arguments as a second args object** (keeping `match.params` as first argument to stay backward compatible).

```js
return store.dispatch(action(match.params, {
  match,
  location,
}));
```
In the next major release we might consider removing `match.params` as argument.

Reply:
Alright, I refactored it.

Reply:
Also updated docs and PR description